stan/math/prim/mat/prob/poisson_log_glm_lpmf.hpp

```diff
@@ -98,7 +97,7 @@ typename return_type<T_x, T_alpha, T_beta>::type poisson_log_glm_lpmf(
   theta.array() += as_array_or_scalar(alpha_val_vec);
 
   Matrix<T_partials_return, Dynamic, 1> theta_derivative
-      = as_array_or_scalar(y_val_vec) - exp(theta.array());
+      = as_array_or_scalar(y_val_vec) - std::exp(theta.array());
```

Comment:
`std::` is inappropriate if this is anything other than `double` arguments . So either `T_partials_return` has to be `double`, or there hsouldn't be a `std::` here.  You may need a `using std::exp` in scope, though.

Reply:
Reverted.